tests/regression/run_regression.py

```diff
@@ -20,62 +20,85 @@
 
 from pandas import read_parquet
 
-ref_path = f"{Path(__file__).resolve().parent}/reference"
-export_path = f"{Path(__file__).resolve().parent}/data_export/core"
+from tests import conftest
 
-references = set(os.listdir(ref_path))
```

Comment:
The bulk of the changes in this file are just indentation into a function. No changes were made to the logic of the previous regression.